subsys/ipc/ipc_service/backends/Kconfig.rpmsg

```diff
@@ -12,4 +12,13 @@ config IPC_SERVICE_BACKEND_RPMSG_WQ_STACK_SIZE
 	  prevent notifying service users about received data from the system
 	  work queue. Size is the same for all instances.
 
+config IPC_SERVICE_BACKEND_RPMSG_SHMEM_RESET
+	bool "Reset shared memory state"
+	default y
+	help
+	  This configuration normalizes the memory state so that the IPC
+	  service is no longer prone to reading status bits from the
+	  previous reset session. The status field of the shared memory
+	  is set to zero.
```

Comment:
what about something along the lines of:

`Some platforms retain the memory content upon reset. This is problematic because the backend expects a zero-ed memory to be able to correctly setup instances and endpoints at init time. When this parameter is set to 'y' the status region of the shared memory is reset on kernel initialization.`

or something like this.

Reply:
Rephrased the prompt according to your suggestion